mitodl/mitxonline
# Fixed user notifications so they are only seen once

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixed #193

#### What's this PR do?
Prevents repeated displays of a user notification that came from the backend

#### How should this be manually tested?
- Enroll a user in a course run via the product detail page
  - When the enrollment completes, you should be redirected to a new page and see a success banner message
- Navigate to the home page right away (with or without dismissing the notification)

You should not see the success message. On the main branch, this message would be shown every time you loaded a new page within the cookie's life span (20 seconds)
